go.mod

```diff
@@ -61,7 +61,7 @@ require (
 	go.opentelemetry.io/collector/exporter/loggingexporter v0.88.0
 	go.opentelemetry.io/collector/exporter/otlpexporter v0.88.0
 	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.88.0
-	go.opentelemetry.io/collector/extension v0.88.0
+	go.opentelemetry.io/collector/extension v0.88.1-0.20231102040825-2e44da36e2c6
```

Comment:
pinned to main branch for now, it will get updates once OTEL does a new release, which should happen before our next release in Dec